README.md

```diff
@@ -43,7 +43,7 @@ resides in). The examples assume your working directory is set to the project ro
 it.
 1. Ensure your `PYTHONPATH` includes your current directory:
     ```shell script
-    export PYTHONPATH=.:$PYTHONPATH
+    export PYTHONPATH="$(pwd):$PYTHONPATH"
```

Comment:
I think it's right to use `$(pwd)` here rather than `.`, since we always want the project root to be part of `PYTHONPATH` whatever the working directory happens to be. Also double-quote it for safety (in case of weird stuff like pathnames with spaces).